backend/runner/observability/runner.go

```diff
@@ -0,0 +1,95 @@
+package observability
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/noop"
+
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+)
+
+const (
+	runnerMeterName              = "ftl.runner"
+	runnerDeploymentKeyAttribute = "ftl.deployment.key"
+	runnerStateNameAttribute     = "ftl.runner.state.name"
+)
+
+type RunnerMetrics struct {
+	meter                  metric.Meter
```

Comment:
Are we using this reference to the `meter`?

Reply:
not outside of init so this has been moved to init as a local var